README.md

```diff
@@ -255,20 +244,20 @@ my-cluster-zookeeper-client   ClusterIP   172.30.241.146   <none>        2181/TC
 my-cluster-zookeeper-nodes    ClusterIP   None             <none>        2181/TCP,2888/TCP,3888/TCP   9d
 ```
 
-The service you're interested in is `my-cluster-kafka-bootstrap:9092`. We will use it to deploy the Data Index Service later.
+The service that you will use to deploy the Data Index Service is `my-cluster-kafka-bootstrap:9092`.
 
-#### Create Required Topics
+#### Creating the required Kafka topics
 
-Having the cluster up and running, the next step is to create the `Kafka Topic`s required by the Data Index Service: 
+After the cluster is running, create the following Kafka Topics that are required by the Data Index Service:
 
 - `kogito-processinstances-events`
-- `kogito-usertaskinstances-events` 
+- `kogito-usertaskinstances-events`
 - `kogito-processdomain-events`
 - `kogito-usertaskdomain-events`
 
-To create the required topics, in the OpenShift Web Console go to the Installed Operators, Strimzi Operator, Kafka Topic tab. 
+For each required Kafka Topic, in the OpenShift Web Console go to **Installed Operators** -> **Strimzi Operator** -> **Kafka Topic** and create the Topic.
```

Comment:
We have to put a note here that:

If this is a development environment, and the user set the Zookeeper and Kafka replicas to 1 to conserve resources, then it should set also the replicas to 1 in each topic (I got errors on topics by letting the '3' replicas)

Reply:
Done. Added the following note:

```
| :information_source: If you are using a development environment and you set the Zookeeper and Kafka replicas to `1` to conserve resources, ensure that you also set the replicas to `1` in each Kafka Topic. The replica setting must match or you might encounter errors with the Kafka Topics. |
| --- |
```